jms/1.7/modules/ROOT/pages/jms-ack.adoc

```diff
@@ -39,10 +88,29 @@ The `ackId` that identifies the Message will then be passed on to the `jms:ack`
 ----
 
 
-=== Manual Session Recovery
+=== Configure Manual Session Recovery
+
+When using the manual acknowledgement mode, all received messages and not acknowledged are not redelivered by the broker.
```

Comment:
"all received messages" is not the actor "using" the mode. We need to reword to be grammatically correct, something like...:
```suggestion
With manual acknowledgement mode, all messages received but not acknowledged are not redelivered by the broker.
```